src/swsssdk/configdb.py

```diff
@@ -93,10 +150,12 @@ def get_config(self):
         hashes = client.keys('*')
         data = {}
         for _hash in hashes:
-            table_name = _hash.split(':', 1)[0]
-            key = _hash.split(':', 1)[1]
-            if not data.has_key(table_name):
-                data[table_name] = {}
-            data[table_name][key] = client.hgetall(_hash)
+            tokens = _hash.split(':', 1)
+            if len(tokens) == 2:
```

Comment:
We do have such data in configdb. One current example is the initialization state of configdb. We might later add similar configdb-internal data that we don't want to be directly readable by user.

---
In reply to: [129188316](https://github.com/Azure/sonic-py-swsssdk/pull/8#discussion_r129188316) [](ancestors = 129188316)